src/inspector/models/general/generalsettingsmodel.cpp

```diff
@@ -58,6 +58,9 @@ void GeneralSettingsModel::requestElements()
     QSet<EngravingItem*> elementsForIsSmallProperty;
 
     for (EngravingItem* element : m_elementList) {
+        if (element->isNote() && toNote(element)->isTrillCueNote()) {
```

Comment:
That's because the trill cue note is small by definition, so the isSmall property can't be edited.

<img width="557" alt="cueSizeExplanation" src="https://github.com/musescore/MuseScore/assets/93707756/455410ae-f891-4ff3-bb43-329b88141b24">

I've added a comment.